setup.py

```diff
@@ -11,8 +11,7 @@
 
 def get_version():
     base = "1.5"
-    day = time.strftime('%Y%m%d', time.localtime())
-    return '%s-dev%s'%(base, day)
+    return '%s-dev' % base
```

Comment:
What the change here for?

Reply:
If the time is included, the resulting dist-info package path for FedLearner will contain the date, causing the path to change with each compilation, which makes it impossible to add to the Gramine configuration.